### ISSUE TITLE
The training process

### ISSUE THREAD

**Issue by nerddd** [javier2491/FaceQnet#2]
Hello, could you provide the process of training?I'm doing face image quality assessment too. But the training didn't converge...Thank u

**Comment by luameows** [comment]
@nerddd How's things going? I met the same situation as u did. I labeled a dataset and used caffe to make regression, however the loss didn't decrease..

**Comment by javier-hernandezo** [comment]
Hello, you can find the details of the training process in the related paper. I think in the paper the optimizer is not mentioned, I used Adam with their default parameters in Keras.

Regards.
